cypress/support/index.js

```diff
@@ -6,32 +6,13 @@ require("cypress-plugin-tab");
 
 const DEBUG_FLAG = false;
 
-// ***********************************************************
-// This example support/index.js is processed and
-// loaded automatically before your test files.
-//
-// This is a great place to put global configuration and
-// behavior that modifies Cypress.
-//
-// You can change the location of this file or turn off
-// automatically serving support files with the
-// 'supportFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/configuration
-// ***********************************************************
-
-// Import commands.js using ES2015 syntax:
-// import './commands';
+export default DEBUG_FLAG;
+Cypress.Screenshot.defaults({ screenshotOnRunFailure: DEBUG_FLAG });
```

Comment:
We could always make this `true` when running in CI so they show in the cypress dashboard e.g.

```js
const DEBUG_FLAG = process.env.CI || false;
```